internal/engine/executor.go

```diff
@@ -407,6 +427,10 @@ func logEval(
 		Str("action", "alert").
 		Str("action_status", string(evalerrors.ErrorAsAlertStatus(params.GetActionsErr().AlertErr))).
 		Msg("result - action")
+
+	// log business logic
+	minderlogger.BusinessRecord(ctx).
+		AddRuleEval(params.Rule.Type, profileName, params)
```

Comment:
It's not going to be that explicit, but I think you already have the rule and profile names through params (the ActionsParams interface)

Reply:
thanks, addressed in the new patch